### PR TITLE
docs(how-to): add graphviz section

### PR DESCRIPTION
**Pull request by cpcloud**
Add graphviz `how-to`.

### CODE REVIEW

**Comment by cpcloud** [comment]
I think this will be harder to find in Analytics. I don't think we can anticipate all reasons why someone might want to visualize an Ibis expression as a graph.

File: docs/how-to/visualization/graphs.qmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+```{python}
+t = ibis.examples.penguins.fetch()
+expr = t.select(lowered=_.species.lower())
+to_graph(expr)
+```
+
+The way to read the graph is from top to bottom.
+
+* The top of the graph contains Ibis operations with no dependents.
+* The bottom of the graph contains Ibis operations with no dependencies.
```
Comment (lostmygithubaccount) [inline docs/how-to/visualization/graphs.qmd:45]: this reads slightly off to me -- "top to bottom", then bullet points for top and bottom followed by the things in the middle -- in my mind would be top, then middle, then bottom, but not a big deal IMO

**Review by lostmygithubaccount — APPROVED** [review]
awesome! one minor comment

File: docs/how-to/visualization/graphs.qmd (hunk 1 above)
Comment (cpcloud) [reply docs/how-to/visualization/graphs.qmd:45]: Yeah, good point. I can rearrange.

Comment (cpcloud) [reply docs/how-to/visualization/graphs.qmd:45]: Fixed!

**Pull request merged**